data_updates/Python/iati_transaction_spec.py

```diff
@@ -781,6 +791,8 @@ def flatten_activities(self, root):
             participating_org_type = "|".join(participating_org_type_list)
             participating_org_role = "|".join(participating_org_role_list)
             participating_org_narrative = "|".join(participating_org_narrative_list)
+            collaboration_type_code =  activity.xpath("collaboration-type/@code")
```

Comment:
This needs to be:
```
collaboration_type_code =  default_first(activity.xpath("collaboration-type/@code"))
```
xpath returns a list of matching XML values, and that `default_first` function I wrote takes a possibly 0 length list and safely returns the first item.

Once that change is made, it works well:
```
analyst_ui=# select distinct(collaboration_type) from repo.iati_transactions limit 5;
                          collaboration_type                           
-----------------------------------------------------------------------
 
 Private Sector Outflows
 Multilateral (inflows)
 Bilateral, core contributions to NGOs and other private bodies / PPPs
 Multilateral outflows
(5 rows)

```